cloudfoundry/server/create.sh

```diff
@@ -30,9 +32,18 @@ cat << EOF >> ./scdf-manifest.yml
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SKIP_SSL_VALIDATION: $SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SKIP_SSL_VALIDATION
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES: $SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_SERVICES: $SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_SERVICES
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES: rabbit,cloud-config-server
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_SERVICES: mysql,cloud-config-server
     MAVEN_REMOTE_REPOSITORIES_REPO1_URL: $MAVEN_REMOTE_REPOSITORIES_REPO1_URL
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_API_TIMEOUT: $SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_API_TIMEOUT
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_API_TIMEOUT: $SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_API_TIMEOUT
+    TRUST_CERTS: $TRUST_CERTS
+    SPRING_PROFILES_ACTIVE: $SPRING_PROFILES_ACTIVE
+    SPRING_CLOUD_DATAFLOW_APPLICATIONPROPERTIES_STREAM_TRUST_CERTS: $TRUST_CERTS
```

Comment:
yes, you got it. but, I haven't figured out which one is actually the right one yet :-)